automation_tools/scripts/ga-migration/utils.py

```diff
@@ -5,13 +5,75 @@
 import os
 import ast
 import json
-import yaml
+import subprocess
 
 import requests
+import yaml
+from jinja2 import Environment, FileSystemLoader
 
 
 logging.basicConfig(level=logging.INFO)
 
+JinjaEnv = Environment(
+    loader=FileSystemLoader('automation_tools/scripts/ga-migration/templates')
+)
+
+
+def get_repo_services(repo_name):
+    """Get repository services."""
+    travis_raw_url = f"https://raw.githubusercontent.com/inveniosoftware/{repo_name}/master/.travis.yml"
+    is_migrated = not requests.get(travis_raw_url).ok
+
+    search_terms = {
+        'postgres': 'db',
+        '- redis': 'cache',
```

Comment:
Why not `redis-service`? e.g. https://github.com/inveniosoftware/invenio-cache/blob/50879bbf441718eebb81a0356852b76590166824/.travis.yml#L24

Reply:
I found **only** `- redis` in other repos like [invenio-indexer](https://github.com/inveniosoftware/invenio-indexer/blob/0b50ed3f8a3ec683ae6a1e697cd04222a0d8c569/.travis.yml#L25) so in the end I think lowest common denominator is that. What do you think?

Reply:
Good enough, thanks!